src/Diagnostics.HealthChecks/Microsoft.Omex.Extensions.Diagnostics.HealthChecks.csproj

```diff
@@ -15,10 +15,6 @@
     <PackageReference Include="Microsoft.ServiceFabric" />
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" />
     <PackageReference Include="Microsoft.Extensions.Http" />
-    <PackageReference Include="Microsoft.Extensions.ObjectPool" />
```

Comment:
Not necessarily related to this package, but in general projects consuming Omex.Extensions might be consuming transitive dependencies, are we discouraging that?

Reply:
I don't believe this was done on purpose (otherwise it should be commented in some way), is that something we'd want to do?